src/app/CommandSender.cpp

```diff
@@ -537,6 +537,18 @@ CHIP_ERROR CommandSender::FinishCommand(FinishCommandParameters & aFinishCommand
     return FinishCommandInternal(aFinishCommandParams);
 }
 
+CHIP_ERROR CommandSender::AddRequestDataInternal(const CommandPathParams & aCommandPath, DataModel::EncodableToTLV & aEncodable,
```

Comment:
nit: EncodeToTLV is const, so we could make aEncodable const ref.
Can we make aAddRequestDataparams also const?

Reply:
Added const to `EncodeToTLV`

aAddRequestDataParams was intentionally left as non-const. At one point we did have CommandSender supply the commandRef, and because the API is trying to make it easier for things to be added in the future (including possible out parameters) we should leave it as non-const IMO